server.py

```diff
@@ -3,21 +3,30 @@
 
 import cherrypy
 
-"""
-This is a simple Battlesnake server written in Python.
-For instructions see https://github.com/BattlesnakeOfficial/starter-snake-python/README.md
-"""
-
+import server_logic
 
 class Battlesnake(object):
+    """
+    This is a simple Battlesnake server written in Python using the CherryPy Web Framework.
+    For instructions see https://github.com/BattlesnakeOfficial/starter-snake-python/README.md
+    """
+
     @cherrypy.expose
     @cherrypy.tools.json_out()
     def index(self):
-        # This function is called when you register your Battlesnake on play.battlesnake.com
-        # It controls your Battlesnake appearance and author permissions.
-        # TIP: If you open your Battlesnake URL in browser you should see this data
+        """
+        This function is called when you register your Battlesnake on play.battlesnake.com
+        See https://docs.battlesnake.com/guides/getting-started#step-4-register-your-battlesnake
+
+        It controls your Battlesnake appearance and author permissions.
+        For customization options, see https://docs.battlesnake.com/references/personalization
+        
+        TIP: If you open your Battlesnake URL in browser you should see this data.
+
+        """
         return {
             "apiversion": "1",
+            "tags": ["python", "starter-project"],  # Optional informational tags to describe your Battlesnake
```

Comment:
This was an idea we had - adding tags via code? The engine appears to be ignoring it, but engine changes would be needed to support it and make it useful.

I'm on the fence about this - we could always add this later when we have engine support.

Reply:
I really like this idea.  It would require a couple extra steps:

* Engine support to send the tags to play on a snake refresh
* Play support to process and apply the tags
* Define a set of rules that describe how we are going to merge tags on a refresh - i.e. does the refresh wipe out existing snake tags and replace them?
* Some kind of https://play.battlesnake.com/references/snake-tags/ page that lists all of the tags in the system and the slugs that users would need to put into this json list

Reply:
I really like the concept